kotlinlib/test/src/mill/kotlinlib/js/KotlinJsKotestModuleTests.scala

```diff
@@ -11,6 +11,7 @@ object KotlinJsKotestModuleTests extends TestSuite {
 
   private val testResourcePath = os.Path(sys.env("MILL_TEST_RESOURCE_DIR")) / "kotlin-js"
   private val testKotlinVersion = "1.9.25"
+  val testKotestVersion = sys.props.getOrElse("TEST_KOTEST_VERSION", ???)
```

Comment:
is this intentional or should we have a keyword here or throw some error instead of non implemented?

Reply:
"not implemented" is "some error". All we want is detect, when the properties isn't set. It results in a test failure. TBH I'm to lazy to maintain a better but longer error message, as we have this pattern in a lot of places.